flag.go

```diff
@@ -595,3 +596,27 @@ func withEnvHint(envVar, str string) string {
 	}
 	return str + envText
 }
+
+func stringifyFlag(f Flag) string {
+	fv := reflect.ValueOf(f)
+	placeholder, usage := unquoteUsage(fv.FieldByName("Usage").String())
```

Comment:
Though this will break existing user created flags which will immediately fail to satisfy that interface (as well as only storing a list of `Flag`s) internally.